src/issues.js

```diff
@@ -61,11 +62,12 @@ function handleIssuesResponse(response) {
 
 // https://developer.github.com/v3/issues/
 // application/vnd.github.symmetra-preview+json
-function getIssues(projectName, repositoryName) {
+function getIssues(projectName, repositoryName, labelFilter) {
   const midFix = `${projectName}/${repositoryName}`;
+  const labelFix = labelFilter ? `?labels=${labelFilter}` : '';
```

Comment:
`midFix` was just something I used since the URL segment was neither at the beginning or the end.  

For something like query parameters, I would probably consider calling it something like `labelsQueryParams`.  If we add additional query parameters, likely we'll want to keep the addition of the `?` part programmatic and not specific to the actual key / value string.